code/src/Components/DogOrCat.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import './dogOrCat.css'
+
+export const DogOrCat = (props) => {
+  const {animalLover, setAnimalLover} = props
+  return (
+    <div className="question">
+            <label>
+              <h3>Are you a dog person or a cat person?</h3>
+                <div className="dropdown">
+                  <select
+                    onChange={event => setAnimalLover(event.target.value)}
+                    value={animalLover}
+                    required
+                    >
+                        <option value= "">Pick one!</option>
```

Comment:
I'm guessing you don't want this to be an option
```suggestion
                        <option value= "" disabled>Pick one!</option>
```